### PR TITLE
Updating server to return full url

### PR DESCRIPTION
**Pull request by patricksheppard-IDR**
Version number needs updating, will do than in a separate commit.

### CODE REVIEW

File: src/main/java/conversion/BuildVuServlet.java [hunk 1]
```diff
@@ -89,13 +89,13 @@ protected void convert(final Individual individual, final Map<String, String[]>
...
-
+            
             ZipHelper.zipFolder(outputDirectory + "/" + fileNameWithoutExt, outputDirectory + "/" + fileNameWithoutExt + ".zip");
 
             final String outputDir = individual.uuid + "/" + fileNameWithoutExt;
-
-            individual.setValue("previewPath", "output/" + outputDir + "/index.html");
-            individual.setValue("downloadPath", "output/" + outputDir + ".zip");
+            
+            individual.setValue("previewPath", contextURL + "/output/" + outputDir + "/index.html");
```
[the remark this answers is not in the document]
Comment (patricksheppard-IDR) [reply src/main/java/conversion/BuildVuServlet.java]: Personally I think that would be a good idea, although we'd obviously have to update all the clients again.

[the remark this answers is not in the document]
Comment (RFoley) [reply src/main/java/conversion/BuildVuServlet.java]: Sounds like a good idea, would make more sense now.

Could even just drop 'path' and have 'preview' & 'download'.

[the remark this answers is not in the document]
Comment (leonatherton) [reply src/main/java/conversion/BuildVuServlet.java]: My preference would be to include Url as it is more descriptive

[the remark this answers is not in the document]
Comment (RFoley) [reply src/main/java/conversion/BuildVuServlet.java]: good point - @patricksheppard-IDR would you like to update?

[the remark this answers is not in the document]
Comment (patricksheppard-IDR) [reply src/main/java/conversion/BuildVuServlet.java]: Will have to work on it tomorrow as it is getting late, but I can update that.

[the remark this answers is not in the document]
Comment (patricksheppard-IDR) [reply src/main/java/conversion/BuildVuServlet.java]: Have updated to url

**Pull request merged**